### DISCUSSION TITLE
https://greendolphindance.com/guestbook/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#6] [General]
# https://greendolphindance.com/guestbook/

Back to the rough ground!

https://greendolphindance.com/guestbook/

**Comment by greendolphindance** [comment]
test

**Comment by greendolphindance** [comment]
test

**Comment by M80-seafever** [comment]
（轻轻地路过并留下了一个表达赞美之情的爪印 <3

**Comment by hanlin-ren** [comment]
除草辣，快两周没更新了😈😈

**Reply by greendolphindance** [reply]
已更新！

**Comment by Joycoooi** [comment]
申请加入友链哈哈哈哈，很有生命力的文章！

**Reply by greendolphindance** [reply]
主页链接速速发来.！

**Comment by ouuan** [comment]
我在 iconfont.cn 找了半天也没找到哪个 RSS 图标被画成了 WiFi（，这是用的哪个（

**Reply by greendolphindance** [reply]
我不知道！这是主题里自带的图标
确实像wifi（。我看看能不能换一个

**Reply by greendolphindance** [reply]
找到了，还真就是wifi图标：

<img width="130" alt="wifi" src="https://github.com/Siricee/hexo-theme-Chic/assets/87912044/e8f1fbab-3abf-4455-bc26-847d713cd796">

已换！

**Comment by David222ddd** [comment]
路过，发现大佬，遂留爪印~

**Comment by leesama** [comment]
加油
